charts/rasa-x/templates/app-deployment.yaml

```diff
@@ -50,10 +50,12 @@ spec:
           httpGet:
```

Comment:
Hmm, we can go further and pass the whole spec for liveness probe and readiness probe, e.g.:
```
readinessProbe: 
{{ toYaml. Values.app.readinessProbe }}
```